src/Interactive/Host/Interactive/Core/InteractiveHost.LazyRemoteService.cs

```diff
@@ -123,7 +123,7 @@ private async Task<InitializedRemoteService> TryStartAndInitializeProcessAsync(C
 
                     return new InitializedRemoteService(remoteService, result);
                 }
-                catch (Exception e) when (FatalError.ReportAndPropagateUnlessCanceled(e))
+                catch (Exception e) when (FatalError.ReportAndPropagateUnlessCanceled(e, cancellationToken))
```

Comment:
>cancellationToken [](start = 89, length = 17)

This might not be correct. `await ExecuteRemoteAsync` above does not take `cancellationToken` - we don't currently support cancellation of the RPC call, but `JsonRpc.InvokeAsync` that we use still claims it may throw `OperationCanceledException`. Perhaps it has its own cancellation token.

Reply:
alright I am going to suppress this case with the explanation given